embark.el

```diff
@@ -838,8 +842,10 @@ first line of the documentation string; otherwise use the word
   (concat ; fresh copy, so we can freely add text properties
    (cond
     ((stringp (car-safe cmd)) (car cmd))
+    ((keymapp cmd)
+     (propertize (if (symbolp cmd) (format "+%s" cmd) "<keymap>")
```

Comment:
What is the plus sign for? Is that a which-key convention?

Reply:
Yes, which-key does this. We could add a defcustom to format this?

Reply:
I guess the plus is OK, which-key is probably the most recognized comencemos for this. I prefer not to add yet another defcustom unless someone really wants it.